aten/src/ATen/native/TypeProperties.cpp

```diff
@@ -68,11 +68,11 @@ static inline ScalarType combine_categories(ScalarType higher, ScalarType lower)
   return lower;
 }
 
-ScalarType result_type(TensorList tensors) {
+ScalarType result_type(const TensorList& tensors) {
```

Comment:
@resistor:
I'm surprised to see this passed as const-ref -- this is just an ArrayRef, right?  (which we don't usually pass by ref because the size is small).

I'm curious: was this intentional or was the intention to avoid a vector copy because our naming is bad and "TensorList" is an ArrayRef but e.g. "variable_list" is a vector?

Reply:
👍